normandy/selfrepair/static/js/normandy_driver.js

```diff
@@ -68,7 +68,20 @@ let Normandy = {
 
     client() {
         return new Promise(resolve => {
-            let client = {};
+            let client = {
+                plugins: {}
+            };
+
+            // Populate plugin info.
+            for (let k = 0; k < navigator.plugins.length; k++) {
+                let plugin = navigator.plugins[k];
```

Comment:
`PluginArray` seems to implement the iterator protocol, so it works fine:

```
>> for (let plugin of navigator.plugins) { console.log(plugin); }
Plugin { application/googletalk: MimeType, description: "Version: 5.41.0.0", filename: "libnpgoogletalk.so", version: "", name: "Google Talk Plugin", length: 1, 1 more… }
Plugin { application/o1d: MimeType, description: "Version: 5.41.0.0", filename: "libnpo1d.so", version: "", name: "Google Talk Plugin Video Renderer", length: 1, 1 more… }
```

I don't know about `Array.from` in Firefox though.